File: Python/1441. Build an Array With Stack Operations.py
class Solution:
    def buildArray(self, target: List[int], n: int) -> List[str]:
        res = []

        for i in range(1, max(target) + 1):
            if i in target: res.append("Push")
            else: res.extend(["Push", "Pop"])
        
        return res

File: Java/1441. Build an Array With Stack Operations.java
import java.util.Arrays;

class Solution {
    private int indexOf(int[] array, int target){
        for(int i = 0; i < array.length; i++){
            if(array[i] == target) return i;
        }
        return -1;
    }

    public List<String> buildArray(int[] target, int n) {
        List<String> ans = new ArrayList<>();

        for(int i = 1; i <= target[target.length - 1]; i++){
            if(indexOf(target, i) != -1) ans.add("Push");
            else{
                ans.add("Push");
                ans.add("Pop");
            }
        }

        return ans;
    }
}
